mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -431,18 +499,29 @@ export const Columns = (props: ColumnsProps) => {
           <VerticalContainer className={styles.cell}>
             {row.original.comments
               .map((comment) => (
-                <>
-                  <Link
-                    value={getName(props.mentors, comment.ownerUuid, ownerName)}
-                    path={pages.user.getPath({uuid: comment.ownerUuid})}
-                  />
-                  <EditableTextarea
-                    text={comment.description}
-                    onChangeFinish={(text) => updateComment(comment, text)}
-                    isEditable={comment.ownerUuid === user?.uuid}
-                    className={styles.editableTextarea}
+                <HorizontalContainer
+                  className={styles.gap}
+                  key={comment.uuid}
+                >
+                  <VerticalContainer>
+                    <Link
+                      value={getName(props.mentors, comment.ownerUuid, ownerName)}
+                      path={pages.user.getPath({uuid: comment.ownerUuid})}
+                    />
+                    <EditableTextarea
+                      text={comment.description}
+                      onChangeFinish={(text) => updateComment(comment, text)}
+                      isEditable={comment.ownerUuid === user?.uuid}
+                      className={styles.editableTextarea}
+                    />
+                  </VerticalContainer>
+                  {comment.ownerUuid === user?.uuid &&
+                  <Button
```

Comment:
Could we use icons instead of buttons? I think buttons could broke whole ui

Reply:
Done